build.sbt

```diff
@@ -31,8 +39,8 @@ ThisBuild / scmInfo := Some(
       "scm:git@github.com:hedgehogqa/scala-hedgehog.git"
     )
   )
-ThisBuild / scalaVersion := "2.12.11"
-ThisBuild / crossScalaVersions := Seq("2.11.12", scalaVersion.value, "2.13.3")
+ThisBuild / scalaVersion := "2.13.3"
+ThisBuild / crossScalaVersions := Seq("2.11.12", "2.12.12", "2.13.3", "3.0.0-M1")
```

Comment:
Should probably make 2.13.3 here `scalaVersion.value`?

Reply:
Ah! Yeah I will fix it. Thanks!